spec/report_spec.rb

```diff
@@ -1092,6 +1092,104 @@ def gloops
     }
   end
 
+  describe "breadcrumbs" do
+    let(:timestamp_regex) { /^\d{4}\-\d{2}\-\d{2}T\d{2}:\d{2}:[\d\.]+Z$/ }
+
+    it "includes left breadcrumbs" do
+      Bugsnag.leave_breadcrumb("Test breadcrumb")
+      notify_test_exception
+      expect(Bugsnag).to have_sent_notification{ |payload, headers|
```

Comment:
Use `do` + `end`, although I suspect there could be some Ruby precedence issue that prevents this from working; if that's the case, add a space before the `{` (same comment applies to a few other lines below)